client/src/components/Details.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReadMore from './ReadMore'
+
+class Details extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showReadMore: false
+    }
+  }
+
+  handleReadMoreClick(event) {
+    event.preventDefault();
+    this.setState({
+      showReadMore: true
+    });
+  }
+
+  render() {
+    return (
+      <a href="https://www.templink.com">Read more about the neighborhood</a>
+    );
+  }
+}
```

Comment:
You should finish conditionally rendering the `ReadMore` component here before merging in this PR. You're incredibly close. See @lndncole's front end code for examples on conditional rendering of a component.